src/crypto/BUILD.gn

```diff
@@ -20,18 +20,35 @@ import("${chip_root}/build/chip/buildconfig_header.gni")
 
 import("crypto.gni")
 
+if (chip_crypto == "") {
+  if (current_os == "android" || current_os == "freertos" ||
```

Comment:
This makes Android crypto mbedtls by default. It should be openssl or boringssl

Reply:
It was copied from `crypto.gni`, so changing it in this PR would be a behavioural change...

Reply:
If you agree to leave it as it was before, please resolve the conversation.